app/jobs/email_published_work_job.rb

```diff
@@ -19,8 +19,8 @@ def perform(publication:, current_user:, previous_event: nil)
     emails << current_user[:email] if current_user
     emails << publication.author_publications.map(&:email)
```

Comment:
add `if Rails.env.production?` so this only gets added for production.. remove the following comment